ilastik/experimental/api/_pipelines.py

```diff
@@ -52,6 +52,21 @@ class PixelClassificationPipeline:
     prob_maps = pipeline.get_probabilities(img)
     ```
 
+    Example using an OME-Zarr dataset stored remotely:
+    ```Python
+    import xarray
+    from ilastik.experimental.api import PixelClassificationPipeline
+    from lazyflow.utility.io_util.OMEZarrStore import OMEZarrStore
+
+    store = OMEZarrStore("<https://example.com/data.zarr>")
+    zarray = store.get_zarr_array("<scale name>")
+    dims = tuple(store.axistags.keys())
+
+    img = xarray.DataArray(zarray, dims=dims)  # downloads the entire image
```

Comment:
The zarray is of course capable of doing things lazily - but this is up to the API user's implementation. I could extend the docstring to mention that you can do e.g. `da = dask.array.from_zarr(zarray)` and then do your thing with `da` if you want to run lazy computations?

Or, you know, we could point them to the lazyflow docs and suggest setting up their own computation graph 😝

Reply:
I was a bit off-point with the above comment; the limitation to laziness here is `get_probabilities`. It only accepts `VigraArray` or `xarray.DataArray`, neither of which support lazy accessing as far as I can tell.

So not a topic for this PR from that point of view, though it would be a great feature.